<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ctkDICOMTableManager</class>
 <widget class="QWidget" name="ctkDICOMTableManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>473</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="headerWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLabel" name="lblPatients">
        <property name="text">
         <string>Patients:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkSearchBox" name="patientsSearchBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblStudies">
        <property name="text">
         <string>Studies:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkSearchBox" name="studiesSearchBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblSeries">
        <property name="text">
         <string>Series:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ctkSearchBox" name="seriesSearchBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="tableSplitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="handleWidth">
      <number>0</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="ctkDICOMTableView" name="patientsTable"/>
     <widget class="ctkDICOMTableView" name="studiesTable"/>
     <widget class="ctkDICOMTableView" name="seriesTable"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkDICOMTableView</class>
   <extends>QWidget</extends>
   <header>ctkDICOMTableView.h</header>
  </customwidget>
  <customwidget>
   <class>ctkSearchBox</class>
   <extends>QLineEdit</extends>
   <header>ctkSearchBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>patientsTable</sender>
   <signal>selectionChanged(QStringList)</signal>
   <receiver>ctkDICOMTableManager</receiver>
   <slot>onPatientsSelectionChanged(QStringList)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>studiesTable</sender>
   <signal>selectionChanged(QStringList)</signal>
   <receiver>ctkDICOMTableManager</receiver>
   <slot>onStudiesSelectionChanged(QStringList)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patientsTable</sender>
   <signal>queryChanged(QStringList)</signal>
   <receiver>ctkDICOMTableManager</receiver>
   <slot>onPatientsQueryChanged(QStringList)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>studiesTable</sender>
   <signal>queryChanged(QStringList)</signal>
   <receiver>ctkDICOMTableManager</receiver>
   <slot>onStudiesQueryChanged(QStringList)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patientsTable</sender>
   <signal>queryChanged(QStringList)</signal>
   <receiver>studiesTable</receiver>
   <slot>onUpdateQuery(QStringList)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>studiesTable</sender>
   <signal>queryChanged(QStringList)</signal>
   <receiver>seriesTable</receiver>
   <slot>onUpdateQuery(QStringList)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>671</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patientsTable</sender>
   <signal>selectionChanged(QStringList)</signal>
   <receiver>studiesTable</receiver>
   <slot>onUpdateQuery(QStringList)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>studiesTable</sender>
   <signal>selectionChanged(QStringList)</signal>
   <receiver>seriesTable</receiver>
   <slot>onUpdateQuery(QStringList)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>502</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patientsTable</sender>
   <signal>selectionChanged(QStringList)</signal>
   <receiver>studiesTable</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>143</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>studiesTable</sender>
   <signal>selectionChanged(QStringList)</signal>
   <receiver>seriesTable</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>227</y>
    </hint>
    <hint type="destinationlabel">
     <x>688</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>seriesSelectionChanged(QStringList)</signal>
  <signal>seriesSelectionChanged(QItemSelection,QItemSelection)</signal>
  <signal>studiesSelectionChanged(QStringList)</signal>
  <signal>studiesSelectionsChanged(QItemSelection,QItemSelection)</signal>
  <signal>patientsSelectionChanged(QStringList)</signal>
  <signal>patientsSelectionChanged(QItemSelection,QItemSelection)</signal>
 </slots>
</ui>
